base/docs/helpdb/Base.jl

```diff
@@ -2603,7 +2603,8 @@ creates a task, and does not run it.
 """
     fld(x, y)
 
-Largest integer less than or equal to `x/y`.
+Largest integer `n` such that `y * n` is less than or equal to `x`. Note that this can be
```

Comment:
depends whether

> the definition of `fld`

was referring to definition as in behavior of the code, or what we document. We'd like them to be the same, but which is the code doing right now?